cypress/support/fragments/finance/funds/funds.js

```diff
@@ -101,6 +101,22 @@ export default {
     cy.expect(Section({ id: transactionResultPaneId }).find(HTML(including(fundCode))).exists());
   },
 
+  transferAmount: (amount, fundFrom, fundTo) => {
+    cy.do([
+      Button('Actions').click(),
+      Button('Transfer').click()
+    ]);
+    cy.expect(Modal('Transfer').exists());
+    cy.do([
+      TextField('Amount*').fillIn(amount.toString()),
+      Selection('From').open(),
+      SelectionList().select((fundFrom.name).concat(' ', '(', fundFrom.code, ')')),
```

Comment:
see the same code in next lines

Reply:
Yes, since we need to fill information for 2 funds

Reply:
I can put a wrapper method for 2 lines of code, but do we really need it since it needed only in this one method?

Reply:
may be one more foreach can be applied?

Reply:
I'd like to leave it as it is since we're entering different data into different fields on this Panel, I think we shouldn't make it more complicated than needed in this particular case.